### ISSUE TITLE
Selecting multiple Bboxs(objects) with the Ctrl key

### ISSUE THREAD

**Issue by jinhojang6** [opencv/cvat#1115]
CVAT currently offers a feature selecting only a single bounding box(object) through a mouse left click.

The thing is allowing annotators to select only one object is not efficient to handle a lot of labels in a frame. If CVAT provides users with a feature to select multiple objects in the annotation panel, it will ramp up the efficiency of the annotation work.

My idea is to enable annotators to choose multiple objects in the annotation panel with the `ctrl` key like what we have been doing to select multiple files on Windows or Mac. After selecting the multiple objects, they can change the label(car, person..) of the selected objects at the same time.

The `Group Shapes` button doesn't support this feature so I'd like to inquire if CVAT can add a feature to improve the efficiency of the work someday.

In a nutshell, from a user's perspective in the screenshot below,

![cvat](https://user-images.githubusercontent.com/41753422/73706175-20f82f80-473b-11ea-8f2c-bc22e8b0defc.PNG)

1. Users can select the 3 people riding a bicycle on the left-hand side using the `Ctrl` key.

2. Users can change the label of the selected 3 objects from `car` to `person` at the same time.

**Comment by TOsmanov** [comment]
@jinhojang6 ,

You can change the label of an object using the keyboard shortcut Ctrl+(1,2,3,4,5,6,7,8,9). I think it will be even faster.
In your case, you need to move the cursor over the object and press Ctrl+(1,2,3,4,5,6,7,8,9) (the number should correspond to the index number of the label "person" in the list of labels) and repeat for all objects with this label.

**Comment by domef** [comment]
This feature would be very helpful!

**Comment by 0xotHik** [comment]
Is that still not implemented? Such needed feature for multiple copypasting :(
Someone found some workaround maybe?

**Comment by kaikicamilo** [comment]
Any tips about selecting a great amount of objects to change their labels? I'm stuck in my project because I need to change like 3000 objects labels and I just can't find a way to select all of them without manual work.

**Comment by kurtjcu** [comment]
+1 to being able to select a bunch of annotations and change a property i.e. change the class

**Comment by royalbonez** [comment]
+1 on this

**Comment by yusufsallam64** [comment]
+1

**Comment by fshaker** [comment]
+1

**Comment by An-u-rag** [comment]
+1

**Comment by vittorio-prodomo** [comment]
+1

**Comment by streamsnipersgames** [comment]
+1

**Comment by heldermtx** [comment]
+1

**Comment by MtTamalpais** [comment]
+100

**Comment by Paul1298** [comment]
+1
